src/Codeception/Lib/Connector/Phalcon34x/MemorySession.php

```diff
@@ -0,0 +1,309 @@
+<?php
+namespace Codeception\Lib\Connector\Phalcon;
```

Comment:
This line should be `namespace Codeception\Lib\Connector\Phalcon34x;`

Reply:
thx :)